### PR TITLE
Fix outdated relref to client-eviction

### PR DESCRIPTION
**Pull request by dzoech**

### CODE REVIEW

File: content/commands/client-no-evict/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ syntax_fmt: CLIENT NO-EVICT <ON | OFF>
 syntax_str: ''
 title: CLIENT NO-EVICT
 ---
-The `CLIENT NO-EVICT` command sets the [client eviction]({{< relref "/develop/connect/clients/#client-eviction" >}}) mode for the current connection.
+The `CLIENT NO-EVICT` command sets the [client eviction]({{< relref "/develop/reference/clients/#client-eviction" >}}) mode for the current connection.
```
Comment (dwdougherty) [inline content/commands/client-no-evict/index.md:46]:
```suggestion
The `CLIENT NO-EVICT` command sets the [client eviction]{{< relref "/develop/reference/clients">}}#client-eviction) mode for the current connection.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Thank you @dzoech ! Much appreciated. Just a couple of comments:

1. Corrected link spec.

1. There's one more occurrence of this link further below. Would you please fix it as well?

**Review by dwdougherty — APPROVED** [review]
LGTM. Thanks again, @dzoech !

**Comment by dzoech** [comment]
Sorry I must have missed the notification about your comment. Thanks for updating.

**Pull request merged**